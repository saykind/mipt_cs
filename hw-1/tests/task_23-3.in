} 77 123.3465
