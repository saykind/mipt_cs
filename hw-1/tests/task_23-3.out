} 77 123.346500